eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Imp/UtxosSpec.hs

```diff
@@ -69,3 +81,22 @@ spec = describe "UTXOS" $
                           (Data $ P.I 32, exUnits)
                       )
               )
+        describe "Scripts pass in phase 2" $ do
+          let scripts' = drop 1 scripts
+          forM_ scripts' $ \(name, script) -> do
+            it name $ do
+              let sHash = hashPlutusScript (script slang)
+              txIn0 <- produceScript sHash
+              exUnits <- getsNES $ nesEsL . curPParamsEpochStateL . ppMaxTxExUnitsL
+              impAnn "Submitting consuming transaction" $
+                submitTx_
+                  ( mkBasicTx mkBasicTxBody
+                      & bodyTxL . inputsTxBodyL .~ Set.singleton txIn0
+                      & isValidTxL .~ IsValid True
+                      & witsTxL . rdmrsTxWitsL
+                        .~ Redeemers
+                          ( Map.singleton
+                              (mkSpendingPurpose $ AsIx 0)
+                              (Data $ P.I 32, exUnits)
+                          )
```

Comment:
You don't need to add redeemers manually. Fixup functionality will take care of it